### PR TITLE
[12.0][mrp_multi_level] define a name_search strategy for the product_mrp_area

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Otherwise the search is not effective.

cc @LoisRForgeFlow

### CODE REVIEW

**Review by HviorForgeFlow — APPROVED** [review]

File: mrp_multi_level/models/product_mrp_area.py [hunk 1]
```diff
@@ -7,12 +7,14 @@
...
 
 from odoo import api, fields, models, _
+from odoo.osv import expression
 from odoo.exceptions import ValidationError
 
 
 class ProductMRPArea(models.Model):
     _name = 'product.mrp.area'
     _description = 'Product MRP Area'
+    _rec_name = "product_id"
```
Comment (LoisRForgeFlow) [inline mrp_multi_level/models/product_mrp_area.py]: A name_get method is already in place. what's the benefit of this?

Comment (LoisRForgeFlow) [reply mrp_multi_level/models/product_mrp_area.py]: https://github.com/OCA/manufacture/pull/446/files#diff-9ca74319d3d61b1d491d69f9c387dba0R131

**Review by LoisRForgeFlow — APPROVED** [review]
I did a little fix to align the search with the display name. LGTM :stuck_out_tongue_closed_eyes:

**Pull request merged**